clients/payment/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@swan-io/payment",
+  "version": "1.3.13",
+  "private": true,
+  "engines": {
+    "node": ">=18.0.0",
+    "yarn": "^1.22.0"
+  },
+  "scripts": {
+    "prepack": "tsc --build",
+    "clean": "tsc --clean"
+  },
+  "browserslist": [
+    ">0.2%",
+    "not op_mini all",
+    "not dead",
+    "not ie <= 11",
+    "safari >= 12"
+  ],
+  "dependencies": {
+    "@0no-co/graphql.web": "1.0.4",
```

Comment:
Update all dependencies using their version numbers in `banking` (as I did an update yesterday)